packages/plugins/i18n/admin/src/utils/data.ts

```diff
@@ -63,7 +63,11 @@ const removePasswordAndRelationsFieldFromData = <
   ) => {
     return Object.keys(datum).reduce((acc, current: keyof TDatum) => {
       // @ts-expect-error – TODO: fix the fact we can't assign this.
-      const attribute = schemum.attributes[current] ?? { type: undefined };
+      const attribute = schemum.attributes[current] ?? { type: undefined, visible: false };
+
+      if (attribute.visible === false) {
+        return acc;
+      }
```

Comment:
seems good! what do you think if we also add a check like 

`if (!attribute) return acc`

Probably gets the same result, but it probably is more correct to just skip if that attribute does not exist